tempest2-testing-internal/src/main/kotlin/app/cash/tempest2/testing/internal/TestUtils.kt

```diff
@@ -40,29 +42,29 @@ private val AWS_CREDENTIALS_PROVIDER = StaticCredentialsProvider.create(
   AwsBasicCredentials.create("key", "secret")
 )
 
+private const val CONNECT_TIMEOUT_MILLIS = 1_000
+
 fun isServerListening(host: String, port: Int): Boolean {
-  var s: Socket? = null
-  return try {
-    s = Socket(host, port)
-    true
-  } catch (e: Exception) {
-    false
-  } finally {
-    if (s != null) try {
-      s.close()
+  return Socket().use {
```

Comment:
the `use` will call `close()` automatically in the `finally` block that used to be here